other-uses.md

```diff
@@ -25,8 +25,9 @@ We’ve focused on tiles, but since OpenStreetMap – uniquely – gives you acc
 * Specialist routing APIs include [CycleStreets cycle routing](https://www.cyclestreets.net/api/) (UK and beyond)
 
 ## Vector map libraries (mobile)
-* Android libraries include the [Mapbox Android SDK](https://www.mapbox.com/android-sdk/), [mapsforge](http://mapsforge.org/), [Nutiteq Maps SDK](https://developer.nutiteq.com/), [Skobbler Android SDK](http://developer.skobbler.com/), and [Tangram ES](https://github.com/tangrams/tangram-es/).
-* iOS libraries include the [Mapbox iOS SDK](https://www.mapbox.com/ios-sdk/), [Nutiteq Maps SDK](https://developer.nutiteq.com/), [Skobbler iOS SDK](http://developer.skobbler.com/), and [Tangram ES](https://github.com/tangrams/tangram-es/).
+* Android libraries include the [MapLibre Android SDK](https://maplibre.org/projects/maplibre-native/), [Mapbox Android SDK (proprietary)](https://www.mapbox.com/android-sdk/), [mapsforge](http://mapsforge.org/), [Nutiteq Maps SDK](https://developer.nutiteq.com/), [Skobbler Android SDK](http://developer.skobbler.com/), and [Tangram ES](https://github.com/tangrams/tangram-es/).
+* iOS libraries include the [MapLibre iOS SDK](https://maplibre.org/projects/maplibre-native/), [Mapbox iOS SDK](https://www.mapbox.com/ios-sdk/), [Nutiteq Maps SDK](https://developer.nutiteq.com/), [Skobbler iOS SDK](http://developer.skobbler.com/), and [Tangram ES](https://github.com/tangrams/tangram-es/).

```

Comment:
mapbox ios (proprietary) too?

Reply:
Mapbox’s libraries aren’t the only proprietary libraries in this list. Why single them out?

Reply:
True, I removed proprietary

Reply:
because they are the only on the list that were open source in the past and then became proprietary, so it's more confusing communication than the other.

Reply:
The whole article could be refactored and have a distinction beteen floss and proprietary. Right now there is none.